plugins/source/gcp/README.md

```diff
@@ -0,0 +1,73 @@
+# GCP Source Plugin
+
+GCP Source plugin for CloudQuery extract configuration from varietly of GCP APIs and loads them to any of the supported CloudQuery Destination plugins.
+
+## Links
+
+- Tables: [./docs/tables/README.md](./docs/tables/README.md)
+- Homepage: [https://cloudquery.io](https://cloudquery.io)
+- Documentation: [https://docs.cloudquery.io](https://docs.cloudquery.io)
+- Discord: [https://cloudquery.io/discord](https://cloudquery.io/discord)
+
+## Install
+
+```bash
+cloudquery gen source gcp
+```
+
+## Libraries in Use
+
+- https://cloud.google.com/go/docs/reference
+- https://github.com/googleapis/google-api-go-client
```

Comment:
This seems to belong in the contributing guide, not sure it belongs so early in the README.md

Reply:
I think it can also be useful for users to understand where is the original documentation (but also for devs)